### PR TITLE
22.3 fb principals without admin update

### PR DESCRIPTION
**Pull request by Ohsudev**
Modified all "assignedto" selection listings input fields that has Task input sections of Prime. Program will be updated to display only current Prime users and groups.

### CODE REVIEW

**Review by kollil — APPROVED** [review]

File: onprc_ehr/resources/queries/onprc_ehr/PrincipalsWithoutAdminUpdate.query.xml [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+        <tables xmlns="http://labkey.org/data/xml">
+            <table tableName="PrincipalsWithoutAdminUpdate" tableDbType="NOT_IN_DB">
+                <columns>
+                    <column columnName="UserId">
+                        <columnTitle>User ID</columnTitle>
+                        <isKeyField>true</isKeyField>
+                    </column>
+                    column columnName="Type"/>
+                    column columnName="FirstName"/>
+                    column columnName="LastName"/>
```
Comment (labkey-martyp) [inline onprc_ehr/resources/queries/onprc_ehr/PrincipalsWithoutAdminUpdate.query.xml]:
```suggestion
                    <column columnName="Type"/>
                    <column columnName="FirstName"/>
                    <column columnName="LastName"/>
```

Comment (labkey-martyp) [reply onprc_ehr/resources/queries/onprc_ehr/PrincipalsWithoutAdminUpdate.query.xml]: Looks like some typos here are preventing your query from validating

**Review by labkey-martyp — COMMENTED** [review]
Hi Raymond, please see the suggested fix to get your query validation working.

File: onprc_ehr/resources/queries/onprc_ehr/PrincipalsWithoutAdminUpdate.query.xml (hunk 1 above)
Comment (Ohsudev) [reply onprc_ehr/resources/queries/onprc_ehr/PrincipalsWithoutAdminUpdate.query.xml]: Thank you Marty for finding the program errors for me.   I will ensure to use the "validating queries" tool within the schema browser the next time as Keith suggested.

Thanks,
Raymond

**Pull request merged**